frost-core/src/frost/identifier.rs

```diff
@@ -19,10 +19,23 @@ impl<C> Identifier<C>
 where
     C: Ciphersuite,
 {
-    // Serialize the underlying scalar.
-    pub(crate) fn serialize(&self) -> <<C::Group as Group>::Field as Field>::Serialization {
+    /// Serialize the identifier using the ciphersuite encoding.
+    pub fn serialize(&self) -> <<C::Group as Group>::Field as Field>::Serialization {
         <<C::Group as Group>::Field>::serialize(&self.0)
     }
+
+    /// Deserialize an Identifier from a serialized buffer.
+    /// Returns an error if it attempts to deserialize zero.
+    pub fn deserialize(
+        buf: &<<C::Group as Group>::Field as Field>::Serialization,
+    ) -> Result<Self, Error<C>> {
+        let scalar = <<C::Group as Group>::Field>::deserialize(buf)?;
```

Comment:
```suggestion

        let scalar = <<C::Group as Group>::Field>::deserialize(buf)?;
```

Reply:
lol, this is against rustfmt

<img width="542" alt="image" src="https://user-images.githubusercontent.com/552961/207157766-453ef971-d8b0-47ff-98ea-2584475960aa.png">